configure

```diff
@@ -25,4 +25,4 @@ if sys.version_info[:2] != (2, 7):
   sys.stderr.write('\n')
   sys.exit(1)
 
-import configure
+import configure
```

Comment:
this should be undone to avoid churn, it'll just come back next time someone uses vim on it

Reply:
Done.